gc/base/ScanClassesMode.hpp

```diff
@@ -36,23 +36,57 @@ class MM_ScanClassesMode : public MM_Base
 	volatile uintptr_t _scanClassesMode;
 
 public:
+	typedef enum {
```

Comment:
No, OMR will compile without gc/base/ScanClassesMode.hpp. I left it there for this commit so that changes could be reviewed in place. I intend to move it to Modron in a different task (integrate concurrent GC with fvtest/gctest).